themes/default/css/_footerContainer.scss

```diff
@@ -1,5 +1,6 @@
 #footerContainer {
-	position: relative;
+	position: fixed;
+	bottom: 0;
```

Comment:
You may need to add a margin to the bottom of the main container since the space is no longer taken up by the footer (because it is `fixed`).  Or maybe there's already a margin there and it's okay.